StripeConnect/StripeConnect/Source/EmbeddedComponentManager.swift

```diff
@@ -98,6 +98,13 @@ public class EmbeddedComponentManager {
         .init(componentManager: self)
     }
 
+    @_spi(DashboardOnly)
+    public func createAccountManagementViewController(
+        collectionOptions: AccountCollectionOptions = .init()) -> AccountManagementViewController {
```

Comment:
not sure yet whether we plan to set account collection options or whether this does anything for Dashboard, but putting it in for now so we can test it